tpp-app/src/onboarding/PeriodStart.js

```diff
@@ -22,7 +23,11 @@ export default function PeriodStart ({ navigation }) {
 
       <TwoButtonContainer>
         <SkipButton title="Skip" onPress={() => navigation.navigate(STACK_SCREENS["Symptoms Choices"])}/>
-        <NextButton title="Next" onPress={() => navigation.navigate(STACK_SCREENS["Symptoms Choices"])}/>
+        <NextButton title="Next" onPress={() => 
+          {
+            OnboardingService.PostInitialPeriodStart(new Date(2011, 11, 30));
```

Comment:
Is there a reason why this date was chosen?

Reply:
This was just me testing hehe it will be replaced once I make the button for picking dates on the frontend!